api/InkPresenter.json

```diff
@@ -0,0 +1,144 @@
+{
+  "api": {
+    "InkPresenter": {
+      "__compat": {
+        "spec_url": "https://wicg.github.io/ink-enhancement/#ink-presenter",
+        "support": {
+          "chrome": {
+            "version_added": "94"
+          },
+          "chrome_android": "mirror",
+          "edge": {
+            "version_added": "93"
```

Comment:
This is weird. How is it that the Edge version is before the Chrome version? (Here and for the other ink).

@queengooborg Can we test this?

Reply:
It is a bit strange...maybe it should just be 94.

Reply:
Microsoft led the development of Ink APIs so Edge getting the feature early actually does make sense.  I think the latest mdn-bcd-collector version has the IDL for this interface, so if desired, we can confirm the data through it!

(By the way, welcome back Chris, it's great to see you again!)

Reply:
> (By the way, welcome back Chris, it's great to see you again!)

👋 ❤️ !

Nice to see you too ;-)